pom.xml

```diff
@@ -1,244 +1,312 @@
+            <url>http://ci.opensecuritycontroller.org:8082/nexus/content/repositories/osc-snapshot</url>
         </repository>
-	</repositories>
+    </repositories>
 
-	<dependencies>
-		<dependency>
-			<groupId>log4j</groupId>
-			<artifactId>log4j</artifactId>
-			<version>1.2.17</version>
-		</dependency>
+    <dependencies>
+        <dependency>
+            <groupId>log4j</groupId>
+            <artifactId>log4j</artifactId>
+            <version>1.2.17</version>
+        </dependency>
 
-		<dependency>
-			<groupId>org.osgi</groupId>
-			<artifactId>org.osgi.service.component.annotations</artifactId>
-			<version>1.3.0</version>
-			<!-- These annotations are build-time only and not needed at runtime. -->
-			<scope>provided</scope>
-		</dependency>
+        <dependency>
+            <groupId>org.osgi</groupId>
+            <artifactId>org.osgi.service.component.annotations</artifactId>
+            <version>1.3.0</version>
+            <!-- These annotations are build-time only and not needed at runtime. -->
+            <scope>provided</scope>
+        </dependency>
 
 		<dependency>
 			<groupId>org.osc.api</groupId>
 			<artifactId>sdn-controller-api</artifactId>
 			<version>2.0.0-SNAPSHOT</version>
 		</dependency>
+      
 
-		<!-- jclouds OpenStack dependencies -->
-		<dependency>
-			<!-- TODO:2.5 Not sure why we need to call this out explicitly... -->
-			<groupId>org.apache.jclouds</groupId>
-			<artifactId>jclouds-core</artifactId>
-			<version>${jclouds.version}</version>
-			<exclusions>
-				<exclusion>
-					<groupId>aopalliance</groupId>
-					<artifactId>aopalliance</artifactId>
-				</exclusion>
-				<exclusion>
-					<groupId>javax.inject</groupId>
-					<artifactId>javax.inject</artifactId>
-				</exclusion>
-			</exclusions>
-		</dependency>
+        <!-- An OSGi bundle version of the AOP dependency -->
+        <dependency>
+            <groupId>org.apache.servicemix.bundles</groupId>
+            <artifactId>org.apache.servicemix.bundles.aopalliance</artifactId>
+            <version>1.0_6</version>
+            <scope>runtime</scope>
+        </dependency>
+        
+        <!-- An OSGi bundle version of the javax.inject dependency -->
+        <dependency>
+            <groupId>org.glassfish.hk2.external</groupId>
+            <artifactId>javax.inject</artifactId>
+            <version>2.5.0-b05</version>
+            <scope>runtime</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.directory.studio</groupId>
+            <artifactId>org.apache.commons.lang</artifactId>
+            <version>2.6</version>
+        </dependency>
+
+        <!-- Declarative Services is used to publish the service at runtime -->
+        <dependency>
+            <groupId>org.apache.felix</groupId>
+            <artifactId>org.apache.felix.scr</artifactId>
+            <version>2.0.4</version>
+            
```

Comment:
this is changed in scope from runtime...any specific reason?

Reply:
Let me see if the project works without these changes